docs/notebooks/survival_model.ipynb

```diff
@@ -0,0 +1,1196 @@
+{
```

Comment:
<p>I think showing a normal likelihood here and using an exponential likelihood below can be confusing. Maybe I'm missing something, let me know</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/bambinos/bambi/pull/742/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='24'/>

Reply:
<p>After second thought, I agree with that. In the first pass, I wanted to implicitly show you didn't have to pass <code>exponential</code> as the likelihood. But it is better to stay consistent with the cat model. </p><p> </p>

Reply:
If I wanted to add a noise term at the observation level to account for variability not explained by $\mu$ (the deterministic part), would that be added to $\lambda$? For example,

$$\alpha \sim \mathcal{N}(0, 1)$$
$$\beta \sim \mathcal{N}(0, 1)$$
$$\epsilon \sim \mathcal{N}(0, 1)$$
$$\mu = \alpha + \beta X$$
$$\lambda \sim \text{Exponential}(\mu)$$
$$Y \sim \lambda + \epsilon$$

Reply:
Good question! Both in Normal and Exponential likelihoods the noise term in the linear predictor is absordeb into the likelihood. 

In the normal case, you say epsilon ~ Normal(0, sigma), consequently, y = mu + epsilon ~ Normal(mu, sigma). Something similar occurs in the Exponential case. So you just need to write

alpha ~ N
beta ~ N
mu = exp(alpha + beta * X)
lambda = 1 / mu
Y ~ Exponential(lambda)

Reply:
Ahh okay. Thanks!!